build/nv.d3.js

```diff
@@ -10000,7 +10000,7 @@ nv.models.lineChart = function() {
                             var point = currentValues[pointIndex];
                             var pointYValue = chart.y()(point, pointIndex);
                             if (pointYValue !== null) {
-                                lines.highlightPoint(i, series.values.indexOf(point), true);
+                                lines.highlightPoint(i, pointIndex, true);
```

Comment:
When looking at https://github.com/novus/nvd3/pull/2054, I can't see this having any harmful side-effects if the new changes "just work". So I'd be inclined to merge this change. We can always revert later if it does break something else, but in the absence of any known defect this looks fine to me.

Reply:
I came to the same conclusion... this seems like it'll help more than it _might_ hurt.